lib/modules/manager/puppet/readme.md

```diff
@@ -1,85 +1,71 @@
-simply keeps Puppetfiles updated
+Renovate can update Puppetfiles.
 
-### How It Works
+### How it works
 
 1. Renovate searches in each repository for any `Puppetfile` files
 1. Existing dependencies are extracted from the relevant sections of the file
 1. Renovate resolves the dependency on the provided forges (or uses `https://forgeapi.puppetlabs.com` as default)
 1. A PR is created with `Puppetfile` updated in the same commit
 1. If the source repository has either a "changelog" file or uses GitHub releases, then Release Notes for each version will be embedded in the generated PR
 
-### supported Puppetfile formats
+### Supported Puppetfile formats
 
-the manager extracts the deps from one Puppetfile
+The `puppetfile` manager extracts the dependencies from one Puppetfile.
+You can define a forge in your `puppetfile` in these ways:
```

Comment:
Is the managers name `puppetfile` or `puppet`?

https://docs.renovatebot.com/modules/manager/puppet/